### PR TITLE
Issue 6 post join search

### LINKED ISSUE #6: Create POST api/search/:id/join/:playerID

### PR DESCRIPTION
**Pull request by JuanMiguelRoman1008**

### CODE REVIEW

**Review by jonfelixrico — COMMENTED** [review]
Same general comment as https://github.com/JuanMiguelRoman1008/discord-utility-bot/pull/9#discussion_r877267727.

File: routes/api/search.js [hunk 1]
```diff
@@ -52,8 +52,30 @@ router.delete("/:id", async (req, res) => {
 // @desc   Manually add someone to your request
 // @access Private
 
-router.post("/:id/player/:playerID", (req, res) => {
-    res.send("Add a player to an LF Request");
+router.post("/:id/player/:playerID", async (req, res) => {
```
[the remark this answers is not in the document]
Comment (JuanMiguelRoman1008) [reply routes/api/search.js:55]: Addressed in 7628ffb0984f38645c01f23d1a9deccd934e0332

File: routes/api/search.js [hunk 2]
```diff
@@ -52,8 +52,30 @@ router.delete("/:id", async (req, res) => {
 // @desc   Manually add someone to your request
 // @access Private
 
-router.post("/:id/player/:playerID", (req, res) => {
-    res.send("Add a player to an LF Request");
+router.post("/:id/player/:playerID", async (req, res) => {
+    try {
+        const id = req.params.id;
```
[the remark this answers is not in the document]
Comment (JuanMiguelRoman1008) [reply routes/api/search.js:57]: Addressed in d8bfdc7d7af71a8c838bd7f2f711f83af723a3cd

[the remark this answers is not in the document]
Comment (jonfelixrico) [reply routes/api/search.js:57]: Seems to be the wrong commit but I'll just do another pass in the other PR/branch

**Pull request merged**